src/service/processing.py

```diff
@@ -45,3 +52,48 @@ def _run_process(
     args: list[Any]
 ):
     asyncio.run(function(data_id, pstorage, args))
+
+
+class Heartbeat:
```

Comment:
Do you think in the future maybe we need something like a `status` method to check the current status of the heartbeat, whether it's running or stopped

Reply:
Based on the current usage there's no need for it, but it'd be easy to add if a use case turns up. Right now it's just start it, forget it, and then stop it when your process is complete

Reply:
👍